packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -570,694 +313,5 @@ class _CupertinoTextSelectionControls extends TextSelectionControls {
   }
 }
 
-// Renders the content of the selection menu and maintains the page state.
```

Comment:
All of the deleted code here and below was moved to packages/flutter/lib/src/cupertino/text_selection_toolbar.dart and sometimes refactored.